index.html

```diff
@@ -387,10 +179,10 @@ <h3>Skills</h3>
           </ul>
         </div>
       </section>
-    </section>
+    </div>
 
     <footer>
-      <section class="form-section">
+      <section class="form-section" id="contact">
```

Comment:
### [OPTIONAL]
- [ ]  I would kindly suggest you increase the width of your contact form(inputs) in the mobile version.
correct this design below 👇  👇 
your design
![image](https://user-images.githubusercontent.com/65068771/146943548-6206c2ce-ee91-48b9-9383-bb5f5e3484a6.png)